src/utils/read-input.js

```diff
@@ -0,0 +1,8 @@
+import { Console } from '@woowacourse/mission-utils';
+import { INPUT_MESSAGE } from '../constants/constants.js';
+
+async function readInput() {
+  return await Console.readLineAsync(INPUT_MESSAGE);
```

Comment:
```suggestion
async function readInput(message) {
  return await Console.readLineAsync(message);
```

`INPUT_MESSAGE`에 의존하지 않고, 해당 함수를 사용하는 곳에서 message를 넘겨주는 형식이면 모듈간 결합도를 개선해 볼 수 있을거 같아요